src/csrc/torchfort.cpp

```diff
@@ -323,3 +362,73 @@ torchfort_result_t torchfort_load_checkpoint(const char* name, const char* check
   }
   return TORCHFORT_RESULT_SUCCESS;
 }
+
+torchfort_result_t torchfort_tensor_list_create(torchfort_tensor_list_t* tensor_list) {
+  *tensor_list = new torchfort::TensorList;
+  return TORCHFORT_RESULT_SUCCESS;
+}
+
+torchfort_result_t torchfort_tensor_list_destroy(torchfort_tensor_list_t tensor_list_in) {
+  auto tensor_list = static_cast<torchfort::TensorList*>(tensor_list_in);
+  delete tensor_list;
+  tensor_list = nullptr;
```

Comment:
do we need this?

Reply:
No we don't, especially since the list is passed in my copy anyway. Removed it.